src/data/real/ennreal.lean

```diff
@@ -1220,11 +1213,12 @@ top_unique $ le_of_forall_nnreal_lt $ λ r hr, h r
 lemma add_div : (a + b) / c = a / c + b / c := right_distrib a b (c⁻¹)
 
 lemma div_add_div_same {a b c : ℝ≥0∞} : a / c + b / c = (a + b) / c :=
-eq.symm $ right_distrib a b (c⁻¹)
+add_div.symm
 
 lemma div_self (h0 : a ≠ 0) (hI : a ≠ ∞) : a / a = 1 :=
 mul_inv_cancel h0 hI
 
+-- *TODO* this should surely be called div_mul_cancel?
 lemma mul_div_cancel (h0 : a ≠ 0) (hI : a ≠ ∞) : (b / a) * a = b :=
```

Comment:
I agree! However I get 295 hits for "mul_div_cancel" in a quick search in VSCode, so let's leave that for another PR :-)

Reply:
I think most of them are not ennreal though :-)